style/swift/sample.swift

```diff
@@ -36,6 +36,15 @@ let cool = yTown(5) { foo in
   }
 }
 
+// Strongify weak references in async closures
+APIClient.getAwesomeness { [weak self] result in
+  guard let `self` = self else { return }
+  if let awesome = result.value {
```

Comment:
Ah sorry I was saying that if we're using "result" in a snippet, we should handle errors. But since error-handling style and eval-weak style in the same snippet is probably a bit too much, maybe just don't use the result, don't _ it even though we should, and then call some really barebones methods that use self.

guard let `self` = self else { return }
self.stopLoadingSpinner()
self.presentAlert()
self.updateDisplay()